bench/gateway-bench.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+const autocannon = require('autocannon')
+
+const query = `query {
+  me {
+    id
+    name
+    nickname: name
+    topPosts(count: 2) {
+      pid
+      author {
+        id
+      }
+    }
+  }
+  topPosts(count: 2) {
+    pid
+  }
+}`
+
+const instance = autocannon(
+  {
+    url: 'http://localhost:3000/graphql',
+    workers: 50,
```

Comment:
drop this workers option, not needed here

Reply:
workers was not removed.

Reply:
my apologies, it's removed locally - just not pushed as the auth doesn't fully work when JIT is enabled and I wanted to avoid a push where the tests are broken (I'll push now so you can see the code!). This is fixed in the Mercurius PR I submitted yesterday :) https://github.com/mercurius-js/mercurius/pull/461. In the future, I'll make sure to only resolve a comment after a push :)